Userland/Utilities/pdf.cpp

```diff
@@ -16,34 +16,51 @@
 #include <LibPDF/Document.h>
 #include <LibPDF/Renderer.h>
 
-static PDF::PDFErrorOr<void> print_document_info_dict(PDF::Document& document)
+static PDF::PDFErrorOr<void> print_document_info(PDF::Document& document, bool json)
 {
+    JsonObject json_output;
+
+    if (!json) {
+        outln("PDF Version: {}.{}", document.version().major, document.version().minor);
+        outln("Number of pages: {}", document.get_page_count());
+    } else {
+        JsonObject version;
+        version.set("major", document.version().major);
+        version.set("minor", document.version().minor);
+        json_output.set("version", version);
+        json_output.set("page_count", document.get_page_count());
+    }
+
     if (auto info_dict = TRY(document.info_dict()); info_dict.has_value()) {
+        auto emit = [&](StringView long_text, StringView json_key, auto value) {
```

Comment:
Can you move the lambda up a bit and also use it for page_count please?

Reply:
done